include/circt/Dialect/RTL/Structure.td

```diff
@@ -61,6 +58,57 @@ def RTLModuleOp : RTLOp<"module",
   let parser = [{ return ::parse$cppClass(parser, result); }];
 }
 
+def RTLExtModuleOp : RTLOp<"extmodule",
```

Comment:
`Ext` is generally short for extension. Do you have any objection to renaming this `ExternModule`?

Reply:
I agree with John: ExternModule seems better here.